index.js

```diff
@@ -170,7 +170,7 @@ function onReady () {
                 client.write(json.targetID, consts.eventNames.sessionEventLoadedExtension)
             })
             break;
-                
+
```

Comment:
can you remove this change from the PR ?

Reply:
Ahhh sorry, didn't see this comment here. I think this is because of `prettier` auto-formatting on save.

Reply:
no worries, those blanks were actually useless 👍